src/class-elements/private-static-method-function-call-access.case

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: PrivateName IdentifierName PropName StringValue
```

Comment:
The description should tell what's being verified in this test, I can't read this and tell anything.